CMakeLists.txt

```diff
@@ -6,6 +6,12 @@ if(NOT CMAKE_CXX_STANDARD OR CMAKE_CXX_STANDARD LESS 17)
   set(CMAKE_CXX_STANDARD 17)
 endif()
 
+if(${CMAKE_VERSION} VERSION_LESS 3.24)
+  set(download_extract_timestamp_flag)
+else()
+  set(download_extract_timestamp_flag DOWNLOAD_EXTRACT_TIMESTAMP TRUE)
```

Comment:
Is this variable used anywhere?

Reply:
Not sure if I understand the question. The dashboard is reporting warnings due to this.

Reply:
The current code only assigns a value to a variable named `download_extract_timestamp_flag`, but I don't see that variable being used anywhere. Is it "magically" picked up by CMake or some other ITK machinery?

I would expect this code here:
```cmake
if(POLICY CMP0135)
	cmake_policy(SET CMP0135 NEW)
endif()
```

Reply:
Not sure about the underlying inner works, just followed the first option the warning mentions as a fix proposal.

Reply:
But you maybe right, as the warnings are still there in the dashboard:
https://open.cdash.org/index.php?project=Insight&date=2023-09-16